src/peer/connection.js

```diff
@@ -161,13 +161,14 @@ class Connection extends EventEmitter {
    * Disconnect from remote peer.
    * @fires Connection#close
    */
-  close() {
+  close(forceClose = false) {
     if (!this.open) {
       return;
     }
+
     this.open = false;
     this._negotiator.cleanup();
-    this.emit(Connection.EVENTS.close.key);
+    this.emit(Connection.EVENTS.close.key, forceClose);
```

Comment:
The `close` event is already used as public event name.
For internal usage, it should be better to use another name as private one like `offer`, `answer`, and `candidate` (= not documented).

Reply:
fixed!